core/isolate.rs

```diff
@@ -402,23 +406,23 @@ impl Isolate {
     out
   }
 
-  pub fn snapshot(&self) -> Result<Snapshot1<'static>, JSError> {
+  pub fn snapshot(&self) -> Result<Snapshot1<'static>, ErrBox> {
```

Comment:
Please add a triple slash comment saying the ErrBox should always be downcast to CoreJsError.